examples/with-msw/pages/index.js

```diff
@@ -1,15 +1,25 @@
 import { useState } from 'react'
 
-export default function Home({ book }) {
+export default function Home({ book, error }) {
   const [reviews, setReviews] = useState(null)
 
   const handleGetReviews = () => {
     // Client-side request are mocked by `mocks/browser.js`.
+    // This request will fail in production, as MSW is only run in development.
+    // In a real-world app this request would hit the actual backend.
     fetch('/reviews')
       .then((res) => res.json())
       .then(setReviews)
   }
 
+  if (error) {
+    return (
+      <div>
+        <p>Failed to fetch book</p>
+      </div>
+    )
+  }
```

Comment:
Here don't handle it as an error, instead render a paragraph that explains why it will only work in localhost, you shouldn't be testing MSW in production.